index.html

```diff
@@ -21,36 +21,54 @@
                
                 <div class="toolbar">
                     <div class="logo"><a href="#">Emmykage</a> </div>
+                  
```

Comment:
- [ ] **Optional:** As the previous reviewer didn't ask for this change, I kept it optional. The left side logo should be the same aligned with the heading of the body. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![n1](https://user-images.githubusercontent.com/70483835/165936351-992c905a-f80c-4339-9982-f5b888c0ba7c.png) | ![n2](https://user-images.githubusercontent.com/70483835/165936366-25a5b9b5-f3ae-4599-b55e-ec2862e52b1f.png)